test/queue_adapters/test_adapter_test.rb

```diff
@@ -1,4 +1,5 @@
 require 'test_helper'
+include ActiveJob::TestHelper
```

Comment:
Please move `include` to inside the test class.

```suggestion
class ActiveJob::Cancel::QueueAdapters::TestAdapterTest< Minitest::Test
  include ActiveJob::TestHelper
```